PowerEditor/installer/nativeLang/brazilian_portuguese.xml

```diff
@@ -360,7 +359,7 @@
                     <Item id="48505" name="Gerar dos arquivos..."/>
                     <Item id="48506" name="Gerar a partir da seleção na área de transferência"/>
                     <Item id="49000" name="Executa&amp;r..."/>
-
+					
```

Comment:
Please restore all the following empty lines (instead of unnecessary white space/TAB).